commands/ssh.go

```diff
@@ -70,14 +64,18 @@ func RunSSH(ns string, out io.Writer) error {
 		return errSSHInvalidOptions
 	}
 
+	// CoreOS has no root user
```

Comment:
I'd prefer not to drop any OS specific code in here. We default to `root`, and if you have to make a change because of an OS requirement, specify it as a parameter.